tools/base/tests/test_runner.py

```diff
@@ -0,0 +1,83 @@
+import importlib
+from unittest.mock import patch, PropertyMock
+
+# import pytest
+
+from tools.base import pytest_runner, runner
+
+
+# this is necessary to fix coverage
```

Comment:
Why? What is going on?

Reply:
Do we have to do this in every `pytest` consuming file?

Reply:
no - only the files that have been imported before pytest is invoked - i expect only the files in this PR will need it - but deps slightly on how we implement in the future 

i have only ever  run into this problem with fixture/pytest plugin files before
/l